framework/Application.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, arg__1, messageHandler, interactionType=Interaction.Yes):
       self.printTag = 'RAVEN Application'
       self.UIs = {}
       self.interactionType = interactionType
-      qtg.QApplication.__init__(self,arg__1)
+      qtw.QApplication.__init__(self,arg__1)
```

Comment:
Unless there's a singular standard for "arg_1", the name should probably be changed to be RAVEN case.